test/visual-diff/visual-tests

```diff
@@ -812,5 +812,16 @@
       "name": "amp-carousel-slides",
       "loading_complete_delay_ms": 2000,
     },
+    {
+      "url": "examples/visual-tests/amp-story/basic-transformed.html",
+      "name": "amp-story: basic example transformed with toolbox-optimizer",
+      "viewport": {"width": 320, "height": 480},
+      "loading_complete_selectors": [
+        ".i-amphtml-story-loaded",
+        "amp-story-page#page-2[active]"
```

Comment:
can we remove the page navigation and its delay? We'd just like to test the render